crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -1183,6 +1218,15 @@ where
     }
 }
 
+#[derive(Debug, Error)]
+pub enum MeshWindingInvertError {
+    #[error("Source mesh does not have primitive topology TriangleList")]
+    WrongTopology,
+
+    #[error("Indices weren't in chunks of 3")]
+    AbruptIndicesEnd,
```

Comment:
We should return information about where the geometry was pathological (and how).

Reply:
That's fine to leave for follow-up then.